api/controllers/controller.go

```diff
@@ -0,0 +1,15 @@
+package controllers
```

Comment:
just noting that the relevant change here is renaming the previous `handlers` package to `controllers`